OpenSim/Tests/Wrapping/testMuscleLengthRegression.cpp

```diff
@@ -0,0 +1,99 @@
+/* -------------------------------------------------------------------------- *
+ *                OpenSim: testMuscleLengthRegression.cpp                     *
+ * -------------------------------------------------------------------------- *
+ * The OpenSim API is a toolkit for musculoskeletal modeling and simulation.  *
+ * See http://opensim.stanford.edu and the NOTICE file for more information.  *
+ * OpenSim is developed at Stanford University and supported by the US        *
+ * National Institutes of Health (U54 GM072970, R24 HD065690) and by DARPA    *
+ * through the Warrior Web program.                                           *
+ *                                                                            *
+ * Copyright (c) 2005-2023 Stanford University and the Authors                *
+ *                                                                            *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may    *
+ * not use this file except in compliance with the License. You may obtain a  *
+ * copy of the License at http://www.apache.org/licenses/LICENSE-2.0.         *
+ *                                                                            *
+ * Unless required by applicable law or agreed to in writing, software        *
+ * distributed under the License is distributed on an "AS IS" BASIS,          *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.   *
+ * See the License for the specific language governing permissions and        *
+ * limitations under the License.                                             *
+ * -------------------------------------------------------------------------- */
+
+#include <OpenSim/OpenSim.h>
+#include <OpenSim/Actuators/ModelOperators.h>
+#include <SimTKcommon.h>
+
+#define CATCH_CONFIG_MAIN
+#include <Vendors/tropter/external/catch/catch.hpp>
+
+using namespace OpenSim;
+
+TimeSeriesTable createMuscleLengthsTable(const Model& model,
```

Comment:
If this is an internal helper function, I'd recommend putting into an anonymous namespace (recommended) or marking with `static` - otherwise, as a reader, I'm wondering whether `createMuscleLengthsTable` is symbol being emitted for consumption via a header in a different translation unit.